examples/engine_cross_compilation.rs

```diff
@@ -97,13 +97,3 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
-
-#[test]
-#[cfg(not(any(
-    windows,
-    // We don't support yet crosscompilation in macOS with Apple Silicon
-    all(target_os = "macos", target_arch = "aarch64")
-)))]
-fn test_cross_compilation() -> Result<(), Box<dyn std::error::Error>> {
-    main()
-}
```

Comment:
I assumed you wanted to remove the `#[cfg]` but keep the `fn test_cross_compilation`, is that correct?